### DISCUSSION TITLE
How to register multiple names

### DISCUSSION THREAD

**Discussion by anoonoll** [react-hook-form/react-hook-form#7985] [Q&A]
I wrote the following code.
Is there any way to register more than one name in the input?

Is there no way...?:_(

```javascript
<input type="text" placeholder="例）120" value={value.time} {...register("time" + value.id, { onChange: (data) => onChange(data, index, "time")})} />
```

**Comment by bluebill1049** [comment]
not sure 100% what do you mean, maybe this is useful? https://react-hook-form.com/api/usefieldarray you shouldn't register same name with different inputs.
